### PR TITLE
Added Identity requirements and harmonized installation description

### PR DESCRIPTION
**Pull request by berndruecker**
This will close https://github.com/camunda-cloud/camunda-cloud-documentation/issues/772

### CODE REVIEW

File: docs/reference/supported-environments.md [hunk 1]
```diff
@@ -33,11 +33,12 @@ We highly recommend running Camunda Platform 8 Self-Managed in a Kubernetes envi
 
 Requirements for the components can be seen below:
 
-|  | Java Version | Elastic |
+|  | Java version | Other requirements |
 |---|---|---|
 | Zeebe Broker and Gateway | OpenJDK 17+ | Elasticsearch 7.16.x(only if Elastic exporter is used) |
 | Operate | OpenJDK 11+ | Elasticsearch 7.16.x |
 | Tasklist | OpenJDK 11+ | Elasticsearch 7.16.x |
+| Identity | OpenJDK 11+ | Keycloak 16.1.1 |
```
Comment (dlavrenuek) [inline docs/reference/supported-environments.md:41]: I forgot to add that Identity is built with Java 17

```suggestion
| Identity | OpenJDK 17+ | Keycloak 16.1.1 |
```

Comment (berndruecker) [reply docs/reference/supported-environments.md]: Good that we take it to a concrete written level :-)

**Comment by akeller** [comment]
@berndruecker I'm going to add my fixes from this page https://c8.docs.camunda.io/docs/self-managed/platform-deployment/ in this PR. I'll also work on fixing the build issues.

**Comment by akeller** [comment]
@berndruecker your build issues are similar to #697. I don't have a better suggestion, so I'm going to do the same thing to get this working and merged. I'll create another issue, or modify the existing one so we remember to fix it.

File: docs/self-managed/platform-deployment/kubernetes-helm.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ By default, the following will be installed:
   - **Zeebe Gateway**: Deploys the standalone Zeebe Gateway with two replicas.
   - **Operate**: Deploys Operate, which connects to an existing Elasticsearch.
   - **Tasklist**: Deploys the Tasklist component to work with user tasks.
+  - **Indentity**: Deploys the Identity component which is responsible for authentication and authorization.
```
Comment (felix-mueller) [inline docs/self-managed/platform-deployment/kubernetes-helm.md:26]: Identity

**Comment by akeller** [comment]
I'm so sorry for all the failed builds here 🙈 I'll get Felix's typo into my next attempt and hopefully we'll be done with this. Redirects are making me crazy!

**Comment by akeller** [comment]
Redirects are painful, but we did it. Since the pages don't exist in production, the redirect was failing. I have an issue to correct this before we push to production - https://github.com/camunda-cloud/camunda-cloud-documentation/issues/697#issuecomment-1093149165

**Comment by akeller** [comment]
@christinaausley can you please review?

**Review by christinaausley — APPROVED** [review]
Some minor phrasing/wording to adjust for either clarity or typos in latest commit. LGTM!

**Pull request merged**